applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_plane_strain_2D_law.cpp

```diff
@@ -11,19 +11,53 @@
 //
 
 // System includes
-#include <iostream>
 
 // Project includes
 #include "custom_constitutive/linear_elastic_plane_strain_2D_law.h"
-
+#include "constitutive_type.h"
 #include "geo_mechanics_application_variables.h"
 
 namespace Kratos
 {
 
+GeoLinearElasticPlaneStrain2DLaw::GeoLinearElasticPlaneStrain2DLaw() = default;
+
+GeoLinearElasticPlaneStrain2DLaw::GeoLinearElasticPlaneStrain2DLaw(std::unique_ptr<ConstitutiveType> pConstitutiveDimension)
+    : GeoLinearElasticLaw{}, mpConstitutiveDimension(std::move(pConstitutiveDimension))
+{
+}
+
+GeoLinearElasticPlaneStrain2DLaw::GeoLinearElasticPlaneStrain2DLaw(const GeoLinearElasticPlaneStrain2DLaw& rOther)
+    : GeoLinearElasticLaw(rOther),
+      mStressVector(rOther.mStressVector),
+      mStressVectorFinalized(rOther.mStressVectorFinalized),
+      mDeltaStrainVector(rOther.mDeltaStrainVector),
+      mStrainVectorFinalized(rOther.mStrainVectorFinalized),
+      mIsModelInitialized(rOther.mIsModelInitialized)
+{
+    if (rOther.mpConstitutiveDimension)
+        mpConstitutiveDimension = rOther.mpConstitutiveDimension->Clone();
+}
+
+GeoLinearElasticPlaneStrain2DLaw& GeoLinearElasticPlaneStrain2DLaw::operator=(GeoLinearElasticPlaneStrain2DLaw const& rOther)
```

Comment:
We always put `const` in front of the type name:
```suggestion
GeoLinearElasticPlaneStrain2DLaw& GeoLinearElasticPlaneStrain2DLaw::operator=(const GeoLinearElasticPlaneStrain2DLaw& rOther)
```

Reply:
Done